ts/hooks/useKeyboardShortcuts.tsx

```diff
@@ -360,6 +361,69 @@ export function useEditLastMessageSent(
   );
 }
 
+export function useStickerChooserShortcut(
+  openStickerChooser: () => unknown
+): KeyboardShortcutHandlerType {
+  const hasOverlay = useHasAnyOverlay();
+
+  return useCallback(
+    ev => {
+      if (hasOverlay) {
+        return false;
+      }
+
+      const key = KeyboardLayout.lookup(ev);
+
+      if (
+        hasExactModifiers(ev, {
+          controlOrMeta: true,
+          shift: true,
+          alt: false,
+        }) &&
+        (key === 'g' || key === 'G')
```

Comment:
See below, how come the usage between `key === 'x' || key === 'X'` and `key.toLowerCase() === 'x'` is inconsistent?